tests/tests.py

```diff
@@ -664,30 +667,6 @@ class TaggableManagerDirectTestCase(TaggableManagerTestCase):
     taggeditem_model = TaggedFood
 
 
-class TaggableManagerDirectCustomPKTestCase(TaggableManagerTestCase):
```

Comment:
I think the test case should remain as it inherits several test functions that we still want to run.

I suppose this highlights the purpose of the TODO comment. It is overriding the test from the base class to avoid a failure. Is that right? If so, maybe it is sufficient to simply remove the word "TODO" but keep the remainder of the comment to explain why the test is skipped.

Reply:
Yes, there were failures when deleting those two tests.